docs/tool_jmap.md

```diff
@@ -0,0 +1,72 @@
+﻿<!--
+* Copyright (c) 2017, 2019 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Java memory map (`jmap`) tool
+
+Use the `jmap` tool to get memory information for a particular Java&trade; process, or list of processes. The tool shows information for every Java process that is owned by the current user ID on the current host. The command syntax is as follows:

```

Comment:
Not sure about `The tool shows information for every Java process that is owned by the current user ID on the current host.`. Should this say the tool *can* show information...? This seems like it was copied from jps, but it doesn't apply to jmap.

Reply:
Hmm ... yes. This was started by someone else last week and I took over yesterday - assumed what was there so far was received as input, although I did strip out some things that didn't seem to apply. Will change to

>The tool shows statistics about classes on the heap, including the number of objects and their aggregate size.